src/ctelnet.cpp

```diff
@@ -1908,3 +1910,34 @@ void cTelnet::raiseProtocolEvent(const QString& name, const QString& protocol)
     event.mArgumentTypeList.append(ARGUMENT_TYPE_STRING);
     mpHost->raiseEvent(event);
 }
+
+// credit: https://github.com/qflow/websockets
+void cTelnet::setKeepAlive(int socketHandle)
+{
+#ifdef Q_OS_WIN
+    struct tcp_keepalive
+    {
+        u_long onoff;
+        u_long keepalivetime;
+        u_long keepaliveinterval;
+    } alive;
+    alive.onoff = TRUE;
```

Comment:
Is there a need for a C `TRUE` from a header file `#define` here - or will the C++ `true` be a better choice?

Reply:
I'm not sure.